wiki/Tournaments/OMIC/2019_4K/en.md

```diff
@@ -2,9 +2,415 @@
+
+| Sunday, 30 June 2019 |  |  |
+| :-: | :-- | :-: |
+| **Grup E** | ::{ flag=ID }:: [MATEH ON](https://osu.ppy.sh/users/8136810) (13 Point), ::{ flag=ID }:: [Noch Einen](https://osu.ppy.sh/users/12668735) (13 Point), ::{ flag=ID }:: [Yuusha006](https://osu.ppy.sh/users/11214161) (10 Point), ::{ flag=ID }:: [THEmpire](https://osu.ppy.sh/users/12142847) (0 Point), ::{ flag=ID }:: [PetrikPintar](https://osu.ppy.sh/users/13489565) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53010360) |
+| **Grup F** | ::{ flag=ID }:: [Merlinch](https://osu.ppy.sh/users/4901971) (13 Point), ::{ flag=ID }:: [Its A Name](https://osu.ppy.sh/users/8724363) (11 Point), ::{ flag=ID }:: [Kurami](https://osu.ppy.sh/users/10184990) (10 Point), ::{ flag=ID }:: [Felter](https://osu.ppy.sh/users/5865319) (2 Point), ::{ flag=ID }:: [Fathan2Gaming](https://osu.ppy.sh/users/11982790) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53006427) |
+| **Grup G** | ::{ flag=ID }:: [alvinrainbow](https://osu.ppy.sh/users/4783042) (15 Point), ::{ flag=ID }:: [anzutto232](https://osu.ppy.sh/users/13408635) (15 Point), ::{ flag=ID }:: [pakyan29](https://osu.ppy.sh/users/7879632) (6 Point), ::{ flag=ID }:: [ilhamsyah849](https://osu.ppy.sh/users/4679215) (0 Point), ::{ flag=ID }:: [\[AST\]Rosary](https://osu.ppy.sh/users/12504843) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53005726) |
+| **Grup H** | ::{ flag=ID }:: [\[-TiaZark-\]](https://osu.ppy.sh/users/14294140) (13 Point), ::{ flag=ID }:: [Dil-](https://osu.ppy.sh/users/3796282) (11 Point), ::{ flag=ID }:: [ReZ\_TB](https://osu.ppy.sh/users/9113955) (6 Point), ::{ flag=ID }:: [Hilmi Ghazian](https://osu.ppy.sh/users/3200342) (6 Point), ::{ flag=ID }:: [RizSama](https://osu.ppy.sh/users/9281315) (0 Point) | [#1](https://osu.ppy.sh/community/matches/53009213) |
+
+## General Rules
+
+- There is no limit to the amount of participants.
+- Participants should register on the links that are already provided.
+- Flag on each participant's osu! account must be **Indonesia**.
+- The lowest rank for this tournament is **#5000 Country**.
+- All of the tournament staff are not permitted to participate as a player on this tournament, except for staff that are taking the position of the following:
+  - Streamer 
+  - Commentator 
+  - Designer 
+- Every participant who registers will go through a session of [Tournament Screening](/wiki/Tournaments/Official_support#tournament-screening).
+
+### Tournament Rules
+
+- Matches will be held every Saturday and Sunday.
+- The scoring system for this tournament is Score V2.
+- Mappool for each round will be **announced by the staff** every Sunday after the previous round is over.
+- All players should **download all maps** for the mappool before the match starts.
+- If there is no referee during the match time, please inform the **organizer staff**.
```

Comment:
```suggestion
- If there is no referee during the match time, players are expected to inform the **organizer staff** in due time.
```

Reply:
(words such as "please inform..." or "please perform..." should be avoided here as generally speaking rules are meant to be firm and binding)